charmcraft.yaml

```diff
@@ -1,6 +1,13 @@
 # Learn more about charmcraft.yaml configuration at:
 # https://juju.is/docs/sdk/charmcraft-config
 type: "charm"
+name: cloudsupport
```

Comment:
These are all keys that used to be provided in metadata.yaml, but can now be put in charmcraft.yaml (as charmcraft can generate the metadata.yaml).  According to https://juju.is/docs/sdk/metadata-yaml , I think it's ok to have both, as long as there aren't the same keys in both.  So `name` should be removed here or from metadata.yaml. :)

Ideally we shouldn't split metadata keys between metadata.yaml and charmcraft.yaml, so perhaps the docs links metadata should go in metadata.yaml for now?  Or migrate everything from metadata.yaml to charmcraft.yaml in the process?

Reply:
Ah right, according to the CI it's even more strict:

> Cannot specify 'links' in charmcraft.yaml when 'metadata.yaml' exists

So we need to either:

1. put all the new docs metadata in metadata.yaml
2. delete metadata.yaml and migrate all the keys to charmcraft.yaml (following the new format; some keys have different names)

Reply:
thank you, I was blindly following the docs guide and wasn't aware of the transition from `metadata.yaml` to `charmcraft.yaml`. I have now moved everything to `charmcraft.yaml`.